File: backend/src/main/java/com/romeo/VetLink/appointment/service/dtos/AppointmentRequestDTO.java
package com.romeo.VetLink.appointment.service.dtos;


import com.romeo.VetLink.patients.service.dtos.PatientDTO;
import lombok.Data;

import java.time.LocalDate;

@Data
public class AppointmentRequestDTO {

    private Integer id;
    private String customerId;
    private String clinicId;
    private String customerEmail;
    private String customerPhoneNumber;
    private String details;
    private PatientDTO patientDTO;
    private LocalDate date;


}

File: frontend/src/app/appointment-request-section/domain/appointment-request-dto.ts
import {PatientDto} from "../../patient-section/domain/patient-dto";

export class AppointmentRequestDto {
  id: number | null = null;
  customerId: string | null = null;
  clinicId: string | null = null;
  customerEmail: string | null = null;
  customerPhoneNumber: string | null = null;
  details: string | null = null;
  patientDTO: PatientDto | null = null;
  patientInfo: String | null = null;

}
